src/game_objects/grid.cpp

```diff
@@ -118,7 +130,8 @@ bool Grid::verify_build(ConstructType type, int col, int row) {
     return candidate->buildable;
 }
 
-Construct* Grid::build(ConstructType type, int col, int row, bool graphical, int id) {
+Construct* Grid::build(ConstructType type, int col, int row, bool graphical,
```

Comment:
Unrelated to PR, but just realized that we don't need the "graphical" bool anymore, since an ID of 0 means that it is on the server, and a non-zero ID means that it's on the client.